### ISSUE TITLE
Uncaught TypeError: q.drain is not a function

### ISSUE THREAD

**Issue by WhoBrokeTheBuild** [dschwen/switch-header-source#13]
[Enter steps to reproduce:]

1. Attempt to swap from a header to a source file
Note: The header and source files are not in the same directory, ex.
    include/app/File.hpp
    src/File.cpp

**Atom**: 1.17.0 x64
**Electron**: 1.4.16
**OS**: linux 4.10.13-1-ARCH
**Thrown From**: [switch-header-source](https://github.com/dschwen/switch-header-source) package 0.22.0

### Stack Trace

Uncaught TypeError: q.drain is not a function

```
At /home/steve/.atom/packages/switch-header-source/node_modules/async/lib/async.js:909

TypeError: q.drain is not a function
    at /packages/switch-header-source/node_modules/async/lib/async.js:909:23
    at /packages/switch-header-source/node_modules/async/lib/async.js:44:16
    at /packages/switch-header-source/node_modules/fs-plus/lib/fs-plus.js:438:24
    at FSReqWrap.oncomplete (fs.js:123:15)
```

### Commands

```
     -0:41.5.0 core:close (atom-pane.pane)
     -0:40.1.0 tree-view:toggle (input.hidden-input)
     -0:38.5.0 settings-view:open (input.hidden-input)
     -0:32.8.0 core:close (div.panels-item)
     -0:31.6.0 switch-header-source:switch (input.hidden-input)
```

### Non-Core Packages

```
atom-hg 2.0.14 
language-batch 0.4.0 
language-cmake 0.3.0 
language-flatbuffers 0.2.0 
language-glsl 2.0.1 
language-powershell 4.0.0 
switch-header-source 0.22.0 
```

**Comment by dschwen** [comment]
Hmm, this seems to pop up in other packages as well. Let me investigate...

**Comment by dschwen** [comment]
Ok, I can reproduce this. After running `npm update` in my local plugin repo I get the async package (which ends up throwing the error).

Linking https://github.com/yongkangchen/remote-sync/issues/374, which is about the same error in a different package

**Comment by dschwen** [comment]
Fix published in v0.22.1
